### PR TITLE
15861 update sentry vue sdk

### PR DESCRIPTION
**Pull request by bolyachevets**
*Issue #: [15861](https://github.com/bcgov/entity/issues/15861)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-filings-dev--pr-492-kfyzkz5g.web.app

File: src/main.ts [hunk 1]
```diff
@@ -54,9 +54,9 @@ async function start () {
     if (sentryDsn) {
       console.info('Initializing Sentry...') // eslint-disable-line no-console
       Sentry.init({
+        Vue,
         dsn: sentryDsn,
         integrations: [
-          new Integrations.Vue({ Vue, attachProps: true }),
           new Integrations.CaptureConsole({ levels: ['error'] })
```
Comment (severinbeauvais) [inline src/main.ts:60]: Is this still the right syntax to add this integration? Can you test that it works?

**Review by severinbeauvais — APPROVED** [review]
LGTM

File: src/main.ts (hunk 1 above)
Comment (bolyachevets) [reply src/main.ts:60]: I tested by adding:
  console.error('HELLO7')
  Sentry.captureException(new Error('HELLO8'))

Both make it to Sentry

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-filings-dev--pr-492-kfyzkz5g.web.app

File: src/main.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/main.ts:60]: Excellent. Thanks!

**Pull request merged**